README.md

```diff
@@ -977,48 +977,20 @@ Here is a demo of an interactive session running on Pixel 5 phone:
 
 https://user-images.githubusercontent.com/271616/225014776-1d567049-ad71-4ef2-b050-55b0b3b9274c.mp4
 
-#### Building the Project using Termux (F-Droid)
-Termux from F-Droid offers an alternative route to execute the project on an Android device. This method empowers you to construct the project right from within the terminal, negating the requirement for a rooted device or SD Card.
-
-Outlined below are the directives for installing the project using OpenBLAS and CLBlast. This combination is specifically designed to deliver peak performance on recent devices that feature a GPU.
-
-If you opt to utilize OpenBLAS, you'll need to install the corresponding package.
+#### Build on Android using Termux (F-Droid)
+F-Droid Termux is an alternative to execute `llama.cpp` on an Android device (no root required).
```

Comment:
I have been looking at the [termux installation instructions](https://github.com/termux/termux-app#installation) and F-Droid is not required, so maybe we could remove the reference to F-Droid here and link to termux instead? Something like this:
```suggestion
#### Build on Android using Termux
[Termux](https://github.com/termux/termux-app#installation) is an alternative to execute `llama.cpp` on an Android device (no root required).
```

Reply:
Thanks for pointing that out, I agree.